src/auto/functions.rs

```diff
@@ -569,111 +573,193 @@ pub fn unix_is_system_fs_type(fs_type: &str) -> bool {
     unsafe { from_glib(gio_sys::g_unix_is_system_fs_type(fs_type.to_glib_none().0)) }
 }
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_at<P: AsRef<std::path::Path>>(mount_path: P) -> (/*Ignored*/UnixMountEntry, u64) {
-//    unsafe { TODO: call gio_sys:g_unix_mount_at() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_at<P: AsRef<std::path::Path>>(mount_path: P) -> (UnixMountEntry, u64) {
+    unsafe {
+        let mut time_read = mem::MaybeUninit::uninit();
+        let ret = from_glib_full(gio_sys::g_unix_mount_at(
+            mount_path.as_ref().to_glib_none().0,
+            time_read.as_mut_ptr(),
+        ));
+        let time_read = time_read.assume_init();
+        (ret, time_read)
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_compare(mount1: /*Ignored*/&mut UnixMountEntry, mount2: /*Ignored*/&mut UnixMountEntry) -> i32 {
-//    unsafe { TODO: call gio_sys:g_unix_mount_compare() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_compare(mount1: &mut UnixMountEntry, mount2: &mut UnixMountEntry) -> i32 {
+    unsafe {
+        gio_sys::g_unix_mount_compare(mount1.to_glib_none_mut().0, mount2.to_glib_none_mut().0)
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//#[cfg(any(feature = "v2_54", feature = "dox"))]
-//pub fn unix_mount_copy(mount_entry: /*Ignored*/&mut UnixMountEntry) -> /*Ignored*/Option<UnixMountEntry> {
-//    unsafe { TODO: call gio_sys:g_unix_mount_copy() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+#[cfg(any(feature = "v2_54", feature = "dox"))]
+pub fn unix_mount_copy(mount_entry: &mut UnixMountEntry) -> Option<UnixMountEntry> {
+    unsafe { from_glib_full(gio_sys::g_unix_mount_copy(mount_entry.to_glib_none_mut().0)) }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//#[cfg(any(feature = "v2_52", feature = "dox"))]
-//pub fn unix_mount_for<P: AsRef<std::path::Path>>(file_path: P) -> (/*Ignored*/UnixMountEntry, u64) {
-//    unsafe { TODO: call gio_sys:g_unix_mount_for() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+#[cfg(any(feature = "v2_52", feature = "dox"))]
+pub fn unix_mount_for<P: AsRef<std::path::Path>>(file_path: P) -> (UnixMountEntry, u64) {
+    unsafe {
+        let mut time_read = mem::MaybeUninit::uninit();
+        let ret = from_glib_full(gio_sys::g_unix_mount_for(
+            file_path.as_ref().to_glib_none().0,
+            time_read.as_mut_ptr(),
+        ));
+        let time_read = time_read.assume_init();
+        (ret, time_read)
+    }
+}
 
-//#[cfg(any(unix, feature = "dox"))]
-//pub fn unix_mount_free(mount_entry: /*Ignored*/&mut UnixMountEntry) {
-//    unsafe { TODO: call gio_sys:g_unix_mount_free() }
-//}
+#[cfg(any(unix, feature = "dox"))]
+pub fn unix_mount_free(mount_entry: &mut UnixMountEntry) {
```

Comment:
This is obviously unsafe :) It should be part of the `Drop` impl of `UnixMountEntry`

Reply:
`UnixMountEntry` used `g_boxed_free`, so I just ignore it